examples/rsp-cra-18/package.json

```diff
@@ -2,8 +2,9 @@
   "name": "rsp-cra-18",
   "version": "0.1.0",
   "private": true,
+  "homepage": ".",
   "dependencies": {
-    "@adobe/react-spectrum": "^3.18.0",
+    "@adobe/react-spectrum": "latest",
     "@spectrum-icons/workflow": "^4.0.0",
```

Comment:
latest for workflow as well?

Reply:
ah yep good idea, that I can bump in the next iteration